frontend/common/Logging.js

```diff
@@ -6,26 +6,16 @@ const log_functions = {
 }
 
 export const handle_log = ({ level, msg, file, line, kwargs }, filename) => {
-    const f = log_functions[level]
+    const f = log_functions[level] || console.log
```

Comment:
BTW, we could try to infer more closely whether it should be one of `debug`, `info`, `warn` and `error` should be used for nonstandard log levels (on the Julia side).  But I figured that that console log isn't the UI we want anyway, so just having a crude fallback to bare `console.log` would be simple for now.